src/inmanta/server/server.py

```diff
@@ -1157,6 +1191,10 @@ def resource_action_update(self, env, resource_ids, action_id, action, started,
 
         if len(messages) > 0:
             resource_action.add_logs(messages)
+            for msg in messages:
+                loglevel = const.LogLevel[msg["level"]].value
+                log_line = data.LogLine.log(loglevel, msg["msg"], *msg["args"])
+                self._write_to_resource_action_log(log_line)
```

Comment:
I'm quite curious about the performance impact of this. 
also about the latency wrt to flushing: does this flush to disk on every write?

Reply:
Yes. The data is flushed to disk on every write. This happens in the StreamHandler class, which is a superclass of WatchedFileHandler (https://github.com/python/cpython/blob/master/Lib/logging/__init__.py).